curbs/README.md

```diff
@@ -317,6 +363,7 @@ A Curb Space is represented as a JSON object whose fields are as follows:
 | `published_date` | [Timestamp][ts] | Required | The date/time that this curb area was first published in this data feed. |
 | `last_updated_date` | [Timestamp][ts] | Required | The date/time that the properties of ths curb area were last updated. This helps consumers know that some fields may have changed. |
 | `curb_zone_id` | [UUID][uuid] | Required | The ID of the Curb Zone this space is within. The geometry of the specified Curb Zone MUST contain the geometry of this space. |
+| `curb_object_ids` | Array of [UUID][uuid] | Conditionally Required | The ID(s) of the [Curb Objects](#curb-object) that this Curb Space is related to. If specified, the objects identified MUST be retrievable through the Curb API. Curb Objects can be related to a Curb Space or a Curb Zone.|
```

Comment:
Similar to the object IDs in a zone, for spaces, this description should specify that the curb objects listed are the objects that are in the space's 'area of influence'. So a meter might be for 2 curb spaces.

Reply:
Addressed in latest commit

Reply:
Resolving so we can review the latest language in a new comment